### PR TITLE
psql_helper in classification_element

### PR DESCRIPTION
**Pull request by bardkw**
In the SMQTK mono-repo from which all this comes from, there was a helper class PsqlConnectionHelper defined in the utilities. This was used in a number of postgresql implementations of things except for some reason the ClassificationElement. We want to use this helper in the ClassificationElement implementation instead of the current state which is basically a re-implementation of that helper.
In the current breakout state of things, the PsqlConnectionHelper class is provided in the smqtk-dataproviders package in the smqtk_dataproviders.utils.postgres module. smqtk-classifier already has a dependency on this package so it should be "free" to use without changing the dependency landscape.

### CODE REVIEW

File: smqtk_classifier/impls/classification_element/postgres.py [hunk 1]
```diff
@@ -237,10 +222,10 @@ def get_classification(self) -> CLASSIFICATION_DICT_T:
             "uuid_val": str(self.uuid)
         }
 
-        conn = self._get_psql_connection()
+        conn = self._psql_helper.get_psql_connection()
```
Comment (Purg) [inline smqtk_classifier/impls/classification_element/postgres.py:225]: Here we want to replace most of body with the use of the `self._psql_helper.single_execute` method. See the `get_bytes` method in `smqtk_dataprovider/impls/data_element/psql.py` for example.

**Review by Purg — CHANGES_REQUESTED** [review]

**Comment by bardkw** [comment]
Altered `get_classification` and `set_classification` to use `single_execute`. The only thing I am unsure about is wether the `ensure_table` function is needed in the callback function.

**Comment by Purg** [comment]
> Altered `get_classification` and `set_classification` to use `single_execute`. The only thing I am unsure about is wether the `ensure_table` function is needed in the callback function.

You do not need to call it manually. As long as `set_table_upsert_sql` is called with a upsert statement string, `ensure_table` will be appropriately called in the `*_execute` methods. Apparently there is a separate issue that `set_table_upsert_sql` is not documented too well (or much at all) but the effect is that if you set an upsert statement string via the `set_table_upsert_sql` method, then it's invocation is set to happen automatically before execution operations.

**Comment by bardkw** [comment]
removed the manual `ensure` calls

**Pull request merged**